spec/helpers/asyncAwait.js

```diff
@@ -0,0 +1,27 @@
+(function(env) {
+  function getAsyncCtor() {
+    try {
+      eval("var func = async function(){};");
```

Comment:
This is nasty but I haven't been able to find a way to do this that neither requires eval nor breaks in environments that don't support async.